PulseEcoSwiftUI/PulseEcoSwiftUI/CityMap/SensorDetails/SensorDView.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  SensorDView.swift
+//  PulseEcoSwiftUI
+//
+//  Created by Maja Mitreska on 1/25/21.
+//  Copyright © 2021 Monika Dimitrova. All rights reserved.
+//
+
+import SwiftUI
+
+struct SensorDView: View {
```

Comment:
Same comment about the name, view models, data, commented out code, view models and logic referenced from the view model, busy body 😊